course_access_groups/security.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+"""
+Course Access Groups permission and authentication classes.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+
+def is_active_staff_or_superuser(user):
+    """
+    Checks if user is active staff or superuser.
+    """
+    return user and user.is_active and (
```

Comment:
Very minor point but why not all on one line?

Reply:
Good question, it doesn't exceeds 79 chars 🤔